tests/test_ddebug_canc/archimedes.c

```diff
@@ -0,0 +1,32 @@
+#include <math.h>
```

Comment:
Binary _archimedes_ should not be included in the PR.

Reply:
Indeed.